### ISSUE TITLE
Make Replace recovery codes keyboard accessible

### ISSUE THREAD

**Issue by ryanfeeley** [mozilla/fxa-content-server#6593]

**Comment by Luna-Coder** [comment]
Hello my name is Erik and I am an Outreachy applicant.
I would like to have this bug assigned to me in order to make my initial contribution.
Thank you.

**Comment by DraciVik** [comment]
Hi @ryanfeeley . I am an Outreachy applicant and I want to contribute to this codebase. May I take on this issue? Thank you!

**Comment by lmorchard** [comment]
@Luna-Coder, looks like you picked up Issue #6154 - do you still want to tackle this one?

@DraciVik, looks like you're taking on Issue #6257 - do you still want to tackle this one?

**Comment by DraciVik** [comment]
@lmorchard Yes I am if @Luna-Coder is ok with it. @Luna-Coder was first to ask so i guess that takes precedence.

**Comment by Luna-Coder** [comment]
@lmorchard I am still working on Issue #6154, but I would also like to tackle this issue. One will be my intial contribution for the application process and plan to use this second issue to get more familiar with the codebase and workflow for this project.

**Comment by lmorchard** [comment]
Let's say this is @Luna-Coder's for now, since @DraciVik has eyes on a couple other issues too.

**Comment by DraciVik** [comment]
@lmorchard. That seems fair. I'll be happy to help as much as I am able. Thank you!

**Comment by AlbionaHoti** [comment]
@lmorchard Due to inactivity, I can continue to work on this issue :)
